src/field_spec.c

```diff
@@ -22,11 +22,6 @@ void FieldSpec_Cleanup(FieldSpec* fs) {
   }
 }
 
-void FieldSpec_Free(FieldSpec* fs) {
```

Comment:
Are you sure we are not using it?

Reply:
Apparently we have FieldSpec_Free() in spec.c